pandas/core/aggregation.py

```diff
@@ -738,7 +738,10 @@ def agg_dict_like(
         if isinstance(selected_obj, ABCDataFrame) and len(
             selected_obj.columns.intersection(keys)
         ) != len(keys):
-            cols = sorted(set(keys) - set(selected_obj.columns.intersection(keys)))
+            cols = sorted(
```

Comment:
can you just use safe_sort?

Reply:
Yes, of course. Thx. Forgot about safe sort...

Reply:
Hm had to add list calls